addons/killers/functions/fnc_createTeleport.sqf

```diff
@@ -0,0 +1,28 @@
+#include "script_component.hpp"
+/*
+ * Author: 3Mydlo3
+ * Function creates teleport to all killers start positions.
+ *
+ * Arguments:
+ * 0: Teleporter <OBJECT>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [_flag] call afsk_killers_fnc_createTeleport
+ *
+ * Public: No
+ */
+
+params ["_flag"];
+
+{
+    private _destinationName = _x getVariable ["LocationName", "Teleport"];
+    _flag addAction [_destinationName, {
+        [QGVAR(teleport), [_this select 0, _this select 1, _this select 3 select 0]] call CBA_fnc_serverEvent;
+        call FUNC(deleteStartPositionsMarkers);
+        removeAllActions (_this select 0);
```

Comment:
`removeAllActions` is evil and should not ever be used. :P
```suggestion
        removeAction (_this select 2);
```

Reply:
But I want to disable teleportation option entirely. So maybe variable on object with array of teleportation action IDs and just remove them in forEach loop?

Reply:
If there is more than one action then this would be a good solution. Or add a variable on object check as action condition and change variable value upon teleportation.